CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.2.7](https://github.com/thecointech/thecoin/compare/v0.2.6...v0.2.7) (2021-06-17)
+
+**Note:** Version bump only for package the-coin
+
```

Comment:
If we adopt a proper template for our commits, this info would be filled in automatically